integration/time_drift/time_drift.bats

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bats
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# This test will make sure that guest time
+# is synced with the host
+
+set -e
+
+load "${BATS_TEST_DIRNAME}/../../lib/common.bash"
+
+# Save guest time
+GUEST_TMP_FILE=$(mktemp guest.XXXXX)
+# Save host time
+HOST_TMP_FILE=$(mktemp host.XXXXX)
+
+function remove_tmp_files() {
+	rm "${GUEST_TMP_FILE}" "${HOST_TMP_FILE}"
+}
+
+trap remove_tmp_files EXIT
```

Comment:
Thanks for adding the trap. It looks like those files are just used to capture the output of `date`, which you then later `cat` back into some shell vars - can we just skip the mktemp's completely, and just put the info direct into shell vars (although, maybe the use of `parallel` and subvars means they will only end up in a subshell, and then get thrown away - let us know...).
I just try to avoid `mktemp` wherever we can as its slower, and more error prone to leaving files laying around in unexpected failure cases.